src/components/schema/SchemaView.js

```diff
@@ -0,0 +1,309 @@
+import React, { Component } from 'react';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import { withStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import SaveIcon from '@material-ui/icons/Save';
+import Alert from '@material-ui/lab/Alert';
+import { Map } from 'immutable';
+import { ColorPicker } from 'material-ui-color';
+import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
+import ReactJson from 'react-json-view';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import { openUrlInBrowser, setSchema } from '../../actions';
+import { JSON_SCHEMA_GETTING_STARTED_URL } from '../../config/constants';
+import {
+  SCHEMA_VIEW_BACK_BUTTON_ID,
+  SCHEMA_VIEW_DESCRIPTION_ID,
+  SCHEMA_VIEW_LABEL_ID,
+  SCHEMA_VIEW_SAVE_BUTTON_ID,
+} from '../../config/selectors';
+import { GRAASP_SCHEMA_ID } from '../../shared/constants';
+import { generateTextColorFromBackground } from '../../utils/color';
+import BackButton from '../common/BackButton';
+import Main from '../common/Main';
+import SchemaTag from '../common/SchemaTag';
+
+const styles = (theme) => ({
+  schemaContent: {
+    padding: theme.spacing(1),
+  },
+  backButton: {
+    position: 'fixed',
+    right: theme.spacing(2),
+    bottom: theme.spacing(2),
+  },
+  title: {
+    marginBottom: theme.spacing(2),
+  },
+  tagWrapper: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+    textAlign: 'center',
+  },
+  tag: {
+    fontSize: 2 * theme.typography.fontSize,
```

Comment:
It's possible that `2em` will be enough, but not sure (basically it's 2 times the inherit font size)